consensus/hotstuff/signature/packer.go

```diff
@@ -229,3 +168,118 @@ func deserializeFromBitVector(serialized []byte, count int) ([]hotstuff.SigType,
 
 	return types, nil
 }
+
+// encodeSignerIndicesAndSigType encodes the given stakingSigners and beaconSigners into signer indices and sig types.
+// The given fullMembers provides the canonical order of signer index for each signer.
+// As an example consider the case where we have a committee C of 10 nodes in canonical oder
+//            C = [A,B,C,D,E,F,G,H,I,J]
+// where nodes [B,F] are stakingSigners and beaconSigners are [C,E,G,I,J].
+//  * First return parameter: QC.signerIndices
+//    - We start with a bit vector v that has |C| number of bits
+//    - If a node contributed either as staking signer or beacon signer,
+//      we set the respective bit to 1:
+//          [A,B,C,D,E,F,G,H,I,J]
+//             ↓     ↓ ↓     ↓ ↓
+//           0,1,1,0,1,1,1,0,1,1
+//    - Lastly, right-pad the resulting bit vector with 0 to full bytes. We have 10 committee members,
+//      so we pad to 2 bytes:
+//           01101110 11000000
```

Comment:
Fyi, this is how bytes look like only for big endian cpus. 
```suggestion
//           01101110 11000000   (if big endianness)
```